docs/components/popovers.md

```diff
@@ -56,32 +56,32 @@ $(function () {
 Four options are available: top, right, bottom, and left aligned.
 
 <div class="bd-example bd-example-popover-static">
-  <div class="popover bs-popover-top">
-    <div class="arrow"></div>
+  <div class="popover bs-popover-top bs-popover-top-docs">
+    <div class="arrow" x-arrow></div>
     <h3 class="popover-title">Popover top</h3>
     <div class="popover-content">
       <p>Sed posuere consectetur est at lobortis. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.</p>
     </div>
   </div>
 
-  <div class="popover bs-popover-right">
-    <div class="arrow"></div>
+  <div class="popover bs-popover-right bs-popover-right-docs">
+    <div class="arrow" x-arrow></div>
```

Comment:
There aren't problems using `x-arrow`, but as @pkozlowski-opensource said, you may want to reduce the points of junction between Bootstrap and Popper.js to avoid to coupling them too much.